src/components/link/common/inline-link.scss

```diff
@@ -0,0 +1,16 @@
+@use '../../core/styles' as sbb;
+
+:host {
+  @include sbb.link-variables--inline;
+
+  display: inline-block;
+}
+
+:host([negative]) {
+  @include sbb.link-variables--inline; // Needed because of specifity
```

Comment:
Unfortunately, it's still needed because some css is now overridden from `:host([negative])` (which before didn't because there was an equal specificity thanks to `:host([variant="..."]`)